appservice/Containerfile

```diff
@@ -2,8 +2,17 @@ FROM docker.io/redhat/ubi9-minimal
 
 # iputils and procps-ng are just for debugging; drop for production
 RUN microdnf install -y python3-pip iputils procps-ng && microdnf clean all
+# cockpit is not available in UBI, install from CentOS 9 stream; c-bridge is just for debugging, drop for production
+RUN printf '[c9s]\nname = C9S\nbaseurl = http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os\ngpgcheck = 0\n' > /etc/yum.repos.d/c9s.repo
+RUN microdnf install --enablerepo=c9s --setopt=install_weak_deps=0 -y cockpit-ws cockpit-bridge && microdnf clean all
```

Comment:
This is of course rather ugly -- it might not be the final solution. We could also install python3-websockets (or websocat) into our official cockpit/ws container -- but that remains to be discussed with the SREs/reviewers for official deployment.

Reply:
Yup, fine for now!